apis/scheduling/config/v1beta2/defaults.go

```diff
@@ -52,6 +52,8 @@ var (
 	}
 
 	defaultEnablePreemption = pointer.Bool(false)
+
+	defaultGangTimeOutSeconds int64 = 120
```

Comment:
in commuity,  the timeout type is *time.Duration, can we follow it?  And I think 120 as default may be a little small,  10 min may be better.

Reply:
1. Got it,10 mins should be fine;
2. The int64 is followed by other Timeout args in Koordinator, this value will be converted into time.Duration when initing the Gang

Reply:
done